### PR TITLE
Feature/us1899 submit trip form

### PR DESCRIPTION
**Pull request by silbermm**

### CODE REVIEW

File: Gateway/crds-angular/Services/PersonService.cs [hunk 1]
```diff
@@ -147,7 +107,7 @@ public Person GetLoggedInUserProfile(String token)
             person.City = contact.City;
             person.State = contact.State;
             person.PostalCode = contact.Postal_Code;
-            person.AnniversaryDate = contact.Anniversary_Date;
+            person.AnniversaryDate = String.Format("{0:MM//yyyy}", contact.Anniversary_Date);
```
Comment (silbermm) [inline Gateway/crds-angular/Services/PersonService.cs]: this shouldn't be needed anymore

File: Gateway/crds-angular/Services/TripService.cs [hunk 2]
```diff
@@ -416,5 +417,126 @@ public bool ValidatePrivateInvite(int pledgeCampaignId, string guid, string toke
...
+            formResponse.FormId = _configurationWrapper.GetConfigIntValue("TripApplicationFormId");
+            formResponse.PledgeCampaignId = dto.PledgeCampaignId;
+
+            formResponse.FormAnswers = new List<FormAnswer>(FormatFormAnswers(dto));
+
+            var formResponseId = _formSubmissionService.SubmitFormResponse(formResponse);
+            return formResponseId;
+        }
+
+        //private void UpdateProfile()
```
Comment (silbermm) [inline Gateway/crds-angular/Services/TripService.cs]: remove comments?

File: crossroads.net/app/common/profile/personal/profilePersonal.controller.js [hunk 3]
```diff
@@ -86,7 +88,7 @@
 
       if (vm.profileData.person.anniversaryDate !== undefined) {
         var mAdate = moment(new Date(vm.profileData.person.anniversaryDate));
-        vm.profileData.person.anniversaryDate = mAdate.format('MM/DD/YYYY');
+        //vm.profileData.person.anniversaryDate = mAdate.format('MM/DD/YYYY');
```
Comment (silbermm) [inline crossroads.net/app/common/profile/personal/profilePersonal.controller.js]: comments

File: Gateway/Crossroads.AsyncJobs/Crossroads.AsyncJobs.csproj [hunk 4]
```diff
@@ -201,8 +202,7 @@
     <Content Include="packages.config" />
     <Content Include="Unity.config">
       <SubType>Designer</SubType>
-    </Content>
-    <Content Include="MinistryPlatform.config">
+    </Content> <Content Include="MinistryPlatform.config">
```
Comment (jpc11623) [inline Gateway/Crossroads.AsyncJobs/Crossroads.AsyncJobs.csproj]: Accidentally put the tag on same line as the ending tag?

File: Gateway/crds-angular.test/Services/ServeServiceTest.cs [hunk 5]
```diff
@@ -20,7 +20,7 @@ public class ServeServiceTest
         private Mock<IContactService> _contactService;
         private Mock<IOpportunityService> _opportunityService;
         private Mock<IAuthenticationService> _authenticationService;
-        private Mock<IPersonService> _personService;
+        private Mock<crds_angular.Services.Interfaces.IPersonService> _personService;
```
Comment (jpc11623) [inline Gateway/crds-angular.test/Services/ServeServiceTest.cs]: Do these need the full namespace?

Comment (IngageMaddoxT) [inline Gateway/crds-angular.test/Services/ServeServiceTest.cs]: There is another iperson service in the translation project

File: crossroads.net/app/trips/signup/page2/signupPage2.directive.js [hunk 6]
```diff
@@ -11,13 +11,128 @@
...
         numberOfPages: '=',
       },
       templateUrl: 'page2/signupPage2.html',
       controller: 'PagesController as pages',
       bindToController: true,
+      link: link
     };
 
+    function link(scope) {
+      scope.tshirtSizes = tshirtSizes();
```
Comment (jpc11623) [inline crossroads.net/app/trips/signup/page2/signupPage2.directive.js]: Are these data lists something that should be moved out to a service, file, table, or something?

File: crossroads.net/app/trips/signup/pages.controller.js [hunk 7]
```diff
@@ -3,42 +3,71 @@
...
 
     function validateProfile(profile, household) {
+      vm.signupService.page1 = {};
+      vm.signupService.page1.profile = profile;
+      vm.signupService.page1.household = household;
+
       handleNext(2);
     }
 
+    function handleNextt(nextPage, target) {
```
Comment (jpc11623) [inline crossroads.net/app/trips/signup/pages.controller.js]: Typo?

File: crossroads.net/app/trips/signup/page2/signupPage2.directive.js (hunk 6 above)
Comment (IngageMaddoxT) [inline crossroads.net/app/trips/signup/page2/signupPage2.directive.js]: another story will address

**Pull request merged**